src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -68,6 +69,9 @@ def post(self, request, project):
 
         if serializer.is_valid():
             alert_rule = serializer.save()
+            metric_alert_rule_created.send_robust(
```

Comment:
Why do we fire this here? Would it make sense to just use `post_save` on `AlertRule`?

Reply:
Would the user be available there? We don't store it on the rule.

Reply:
Ahh, I missed that... You can use 
```
from sentry.app import env
env.request
```

`env` stores the request in a global place. You'll need to check that the `request` isn't `None` in the signal handler.

Reply:
I'm personally in favor of just keeping it simple like this. I actually didn't know about `env.request`